pyccel/ast/operators.py

```diff
@@ -494,6 +494,14 @@ def __new__(cls, arg1, arg2, simplify = False):
         if simplify:
             if isinstance(arg2, PyccelUnarySub):
                 return PyccelMinus(arg1, arg2.args[0], simplify = True)
+            elif isinstance(arg1, Literal) and isinstance(arg2, Literal):
+                dtype, precision = cls._calculate_dtype(arg1, arg2)
+                if dtype is NativeInteger():
+                    return LiteralInteger(arg1.python_value + arg2.python_value, precision=precision)
```

Comment:
We should be careful here. Currently to tell if an integer is negative we check if `isinstance(a, PyccelUnarySub)`, so a `LiteralInteger` should always be positive (we need to add a check for this to enforce it). I think we should add a test here so:
```
result = arg1.python_value + arg2.python_value
if result >= 0:
    return LiteralInteger(result, precision=precision)
else:
    return PyccelUnarySub(LiteralInteger(-result, precision=precision))
```

Reply:
done in [d289729](https://github.com/pyccel/pyccel/pull/797/commits/d289729fd8de96a2f0200dea18fbb6e770bbf690)